src/openrct2/interface/Viewport.cpp

```diff
@@ -1,4 +1,4 @@
-﻿/*****************************************************************************
+/*****************************************************************************
```

Comment:
I'm not sure what this change is about, locally it doesn't exist.

Reply:
Byte order mark?

Reply:
Hm apparently clang-format is doing this, I have no control over it as any other clang version is not really working.

Reply:
Load it up in a hex editor and it should become clear very soon.

Reply:
It is BOM but that is due to clang-format modifying it, initially its just UTF8 after clang-format its UTF8-BOM

Reply:
How are you using clang-format? Doesn't that output to stdout therefore leaving it to the caller to handle?